File: demo/src/main/java/com/example/demo/gitHubData/model/File.java
package com.example.demo.gitHubData.model;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class File {

    private String sha;

    private String filename;

    private String status;

    private int additions;

    private int deletions;

    private int changes;

    private String blob_url;

    private String raw_url;

    private String contents_url;

    private String patch;
}

File: frontend/src/components/File/File.js
export default class File {
    constructor(sha, filename, status, additions, deletions, changes, blob_url, raw_url, contents_url, patch) {
        this.sha = sha;
        this.filename = filename;
        this.status = status;
        this.additions = additions;
        this.deletions = deletions;
        this.changes = changes;
        this.blob_url = blob_url;
        this.raw_url = raw_url;
        this.contents_url = contents_url;
        this.patch = patch;
    }
}